soweego/linker/classifiers.py

```diff
@@ -18,14 +17,11 @@
 
 
 class SVCClassifier(SKLearnAdapter, BaseClassifier):
-    def __init__(self, *args, kernel='linear', probability=True, **kwargs):
+    def __init__(self, *args, probability=True, **kwargs):
         super(SVCClassifier, self).__init__()
 
-        # Add 'kernel' argument to `kwargs`
-        kwargs.update({'kernel': kernel,
-                       'probability': probability})
+        kwargs['probability'] = probability
```

Comment:
Shouldn't it be possible to set this value via a _cli_ argument? Currently the value gets overwritten regardless of whether it was specified or not.

```suggestion
        kwargs['probability'] = kwargs.get('probability', True)
```